cli/src/funTest/assets/git-repo-expected-output.yml

```diff
@@ -41,7 +41,8 @@ repository:
       url: "https://github.com/dcodeIO/long.js.git"
       revision: "941c5c62471168b5d18153755c2a7b38d2560e58"
       path: ""
-  config: {}
+  config:
+    package_snippet_choices: []
```

Comment:
Just like the other `config` items, we should not serialize `package_snippet_choices` if empty.